client/src/containers/Authentication.jsx

```diff
@@ -0,0 +1,46 @@
+import React, { useState } from "react";
+import { Menu } from "semantic-ui-react";
+import SigninForm from "./SigninForm";
```

Comment:
You can create a custom Menu component and then import from there. 
I would suggest wrapping up the `semantic-UI-react` dependency in one file and extending from there.